converter/internal/prometheusconvert/testdata/ec2.yaml

```diff
@@ -0,0 +1,19 @@
+scrape_configs:
+  - job_name: "prometheus1"
+    static_configs:
+      - targets: ["localhost:9090"]
+    ec2_sd_configs:
+      - region: 'us-east-1'
+        access_key: 'YOUR_ACCESS_KEY'
+        secret_key: 'YOUR_SECRET_KEY'
+        port: 8080
```

Comment:
Do we have any of these tests which validate that having multiple instances of the same SD generates a unique component name?